.github/workflows/basic-build-ci.yaml

```diff
@@ -5,12 +5,18 @@ on:
   - push
 
 jobs:
-  build-dashing:
-    runs-on: ubuntu-latest
+  build-ros:
     strategy:
       fail-fast: false
+      matrix:
+        image: ["rostooling/setup-ros-docker:ubuntu-bionic-ros-dashing-ros-base-latest"]
```

Comment:
I think this structure is not right, if I'm reading https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations correctly believe it should be:

```yaml
      matrix:
        ros-version: [dashing, foxy]
        include:
            - ros-version: dashing
              image: rostooling/setup-ros-docker:ubuntu-bionic-ros-dashing-ros-base-latest
            - ros-version: foxy
              image: rostooling/setup-ros-docker:ubuntu-focal-ros-foxy-ros-base-testing-latest
```

so that when the ROS version is dashing, its respective image will be used, and same goes for foxy.